doc/rfc/0000-rfc-repository.md

```diff
@@ -0,0 +1,168 @@
+# RFC 0000 - RFC repository
+
+- Date proposed: 2023-02-28
+- Date approved: under review
+
+## Summary
+
+This RFC proposes storing OpenTitan RFCs as files in a Git repo, and using
+an open source development workflow for the RFC process. RFCs would be
+stored in the RFC repo as text files. RFCs would be proposed, reviewed, and
+accepted/rejected using GitHub pull requests (PRs). The goal is to ensure
+project decisions are better documented, and to make the RFC process more
+visible and transparent. The workflow proposed here will already be familiar to
+open source contributors and should add little overhead.
+
+## Motivation
+
+RFCs in OpenTitan are used for decisions that need consensus from the technical
+committee. A summary of the RFC process as documented [here][rfc-process]:
+
+1. Write up your RFC somewhere.
+2. Create an issue in the OpenTitan repo with a link to your RFC.
+3. Label the issue with `RFC:Proposal`, then `RFC:TC review` when ready.
+4. Pending RFCs will be discussed in the next technical committee meeting.
+5. Approved RFCs have their label changed to `RFC:Approved`.
+6. Rejected RFCs have their label removed.
+
+Typically, authors write their RFCs in one of two places:
+
+* A Google Doc, accessible only to select people.
```

Comment:
You're right, but I think it's important to say that letting authors choose where they store their RFC means we can't consistently apply a visibility rule.

I've reworded to make this clear, since I think it's relevant to the RFC.